WcaOnRails/app/helpers/delegates_helper.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+module DelegatesHelper
+  def delegate_class(code)
+    {
+      'board_member' => 'alert-success',
+      'candidate_delegate' => 'alert-warning',
+      'senior_delegate' => 'alert-info',
+    }[code]
```

Comment:
I feel like we could avoid this class computation by using a delegate status as the class and adding CSS like
```sass
.board_member {
  @extend .alert-success;
}
.candidate_delegate {
  @extend .alert-warning;
}
.senior_delegate {
  @extend .alert-info;
}
```